plutus-core/cost-model/data/models.R

```diff
@@ -1,72 +1,88 @@
-# Suppress some annoying warnings
+## Suppress some annoying warnings
```

Comment:
why the double hashes?

Reply:
Because emacs [indents single-hash comments to the 40th column](http://ess.r-project.org/Manual/ess.html#Indenting) when you press `return`, which is really annoying.  I see you can turn that off though, so that might be better.